codebuild/spec/sidetrail/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:14.04
+
+RUN apt-get -y install software-properties-common
+RUN apt-add-repository ppa:brightbox/ruby-ng
+RUN apt-get update
+RUN apt-get -y install libssl-dev python-pip ruby2.3
+RUN gem install bundler
+
+ADD codebuild codebuild
+RUN codebuild/bin/install_sidetrail_dependencies.sh
+RUN mkdir -p /sidetrail-install-dir
+RUN codebuild/bin/install_sidetrail.sh /sidetrail-install-dir
+
+ADD . s2n
```

Comment:
CodeBuild will clone the s2n source. While this step will help with local development, we could accomplish the same thing with a docker run argument: `docker run -tiv /home/${USER}/s2n:/s2n smack `.